.eslint-ratchet-high-water-mark

```diff
@@ -1 +1 @@
-2059
+1530
```

Comment:
You might need to get latest and do an npm install. I think Marc updated the eslint version which caused the spike in errors.